Documentation/howto/ovs-with-p4-executables.rst

```diff
@@ -661,6 +661,33 @@ previously configured CONFIG params.
     tdi-portin-id: Port ID for Pipeline in Input Direction
     tdi-portin-id: Port ID for Pipeline in Output Direction
 
+10) Get atrributes for port statistics retrieval::
```

Comment:
Get port statistics for VHOST, Physical link and non-control TAP ports?

Reply:
Done. Taken care.

Reply:
Done